src/components/profile/editProfile/linked-auths.jsx

```diff
@@ -1,21 +1,35 @@
 import React from "react";
-import { externalLoginTypeEnum } from "../../../constants/externalLoginTypeEnum";
+import PropTypes from "prop-types";
+import externalLoginTypeEnum from "../../../constants/externalLoginTypeEnum";
+
+const renderType = type => {
+  switch (type) {
+    case externalLoginTypeEnum.Google:
+      return <i className="fab fa-google" />;
+    case externalLoginTypeEnum.Facebook:
+      return <i className="fab fa-facebook" />;
+    case externalLoginTypeEnum.Twitter:
+      return <i className="fab fa-twitter" />;
+    default:
+      return <i className="fas fa-at" />;
+  }
+};
 
 const LinkedAuths = props => {
   const { item } = props;
   return (
     <div>
       <div className="btn-group m-1" role="group" disabled>
         <a
-          href="#"
+          href="/#"
```

Comment:
Just a curious why it was changed? Which is the eslint rule?

Reply:
> Just a curious why it was changed? Which is the eslint rule?

The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles.